charts/gardener/controlplane/values.yaml

```diff
@@ -17,12 +17,16 @@ global:
       tag: latest
       pullPolicy: IfNotPresent
     livenessProbe:
+      # health probes should be disabled for debugging purposes only
+      disable: false
```

Comment:
From an API design perspective, I would always go for positive expressions to prevent confusion. I.e., I would name this field `enable` and default it to `true`.
But that is in fact a matter of personal preference.